scripts/v2/cipool/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail # fail on...failures
+set -x # log commands as they run
+
+# Note: You must run az login before executing this script
+
+[[ -z "${AZURE_SUBSCRIPTION_ID}" ]] && echo "AZURE_SUBSCRIPTION_ID is not set" && exit 1
+
+ROOT=$(dirname "${BASH_SOURCE[0]}")
+
+
+RESOURCE_GROUP="1es-pool"
```

Comment:
Should this be `aso-1es-pool` or is the Resource Group shared?

Reply:
It's in the ASO subscription, and not shared. I have already created it though so don't necessarily want to delete it and recreate it. Thoughts?